.github/workflows/codeql.yaml

```diff
@@ -0,0 +1,50 @@
+name: "CodeQL Advanced"
+
+on:
+  push:
+    branches: ["main"]
```

Comment:
We do not have this branch

Reply:
Sorry about that; I had just copied the file from the rancher-desktop repo and removed the typescript-javascript setup, and missed this.